src/point_one/fusion_engine/messages/measurements.h

```diff
@@ -1130,17 +1130,17 @@ struct P1_ALIGNAS(4) DeprecatedVehicleSpeedMeasurement : public MessagePayload {
 ////////////////////////////////////////////////////////////////////////////////
 
 /**
- * @brief Raw (uncorrected) heading sensor measurement output (@ref
- *        MessageType::RAW_HEADING_OUTPUT, version 1.0).
+ * @brief Heading sensor measurement output with heading bias corrections
+ *        applied (@ref MessageType::HEADING_OUTPUT, version 1.0).
  * @ingroup measurement_messages
  *
- * This message contains raw heading sensor measurements that have not been
- * corrected for mounting angle biases.
+ * This message is an output from the device contaning heading sensor
+ * measurements that have not been corrected for horizontal and vertical biases.
```

Comment:
Maybe I'm misunderstanding this, but "measurements that have not been corrected for horizontal and vertical biases" seems to contradict "Heading sensor measurement output with heading bias corrections applied".

Reply:
Oops good catch. Copy paste error. Changed to:
```
 * This message is an output from the device contaning heading sensor
 * measurements after applying horizontal and vertical bias corrections to
 * account for the orientation of the primary and secondary GNSS antennas.
 * ```